lib/coffee/AccessTokenEncryptor.coffee

```diff
@@ -1,10 +1,14 @@
 crypto = require('crypto')
+async = require('async')
```

Comment:
```suggestion
```

It was missing in the `dependencies` previously, so there is no need for another deps update.

Reply:
:+1: thanks